testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimatedCubeController.cs

```diff
@@ -13,32 +13,34 @@ public class AnimatedCubeController : NetworkBehaviour
         private bool m_Rotate;
         private NetworkAnimator m_NetworkAnimator;
         private bool m_IsServerAuthoritative = true;
+        public bool IsServerAuthoritative => m_IsServerAuthoritative;
 
-        private void Awake()
+        public override void OnNetworkSpawn()
         {
+            if (HasAuthority())
+            {
+                enabled = false;
+            }
             m_Animator = GetComponent<Animator>();
-            m_NetworkAnimator = GetComponent<NetworkAnimator>();
+            m_NetworkAnimator = GetComponent<OwnerNetworkAnimator>();
```

Comment:
The fact that the order was wrong, combined with the low readability (assigning different component to the same member variable?!) points to the need for a comment. :-)

Maybe something above saying "We want to find either the OwnerNetworkanimator or the regular NetworkAnimator, with preference for the OwnerNetworkanimator".

Just a suggestion.